docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -113,6 +106,40 @@ Because no invocations of the indicated method were found, the method is unneces
     - protected, public, or global instance methods
     - static methods of any visibility
 
+## UnimplementedTypeRule
+
+UnimplementedTypeRule detects abstract classes and interfaces that are non-global and missing implementations or extensions.
+
+To invoke UnimplementedTypeRule, provide `--engine sfge` and `--projectdir/-p` on `scanner:run`.
+
+### Definition
+
+UnimplementedTypeRule is a traditional static analysis rule where a violation occurs at a point in the code where the interface or abstract class is declared. It doesn’t use sources or sinks.
```

Comment:
Should the definition be worded closer to the one for `UnusedMethodRule`?

Reply:
(already peer-reviewed - no changes to make)